CorsixTH/Lua/humanoid_actions/seek_reception.lua

```diff
@@ -75,7 +75,7 @@ local function action_seek_reception_start(action, humanoid)
     local dist = humanoid.world:getPathDistance(humanoid.tile_x, humanoid.tile_y, x, y)
     -- TODO: Make 'else' method work for all humanoids
     if class.is(humanoid, Vip) or class.is(humanoid, Inspector) then
-      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions_on_my_way_to
+      humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions.on_my_way_to
       :format(best_desk.object_type.name))
```

Comment:
Our coding convention mentions that this line should have been indented: https://github.com/CorsixTH/CorsixTH/wiki/Coding-Conventions